spec/rubocop/cop/style/symbol_array_spec.rb

```diff
@@ -201,7 +201,7 @@
     it 'registers an offense for array starting with %i' do
       expect_offense(<<~RUBY)
         %i(one two three)
-        ^^^^^^^^^^^^^^^^^ Use `[:one, :two, :three]` for an array of symbols.
+        ^^^^^^^^^^^^^^^^^ Use array brackets `[...]` for an array of symbols.
```

Comment:
I think the offense messages are less clear here than they used to be. Can we continue showing the full replacement if the array is on a single line?

Also, `Use an array literal […]` would probably be clearer wording for when we cannot show the whole array.

Reply:
Indeed, that's true 👍 
I've made the changes exactly as they should be, so take another look at it.

Reply:
Thanks!